File: FrontEnd/src/app/features/reservations/ledgers/classes/view-models/list/ledger-vm.ts
import { LedgerPortGroupVM } from './ledger-port-group-vm'
import { LedgerReservationVM } from './ledger-reservation-vm'
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface LedgerVM {

    customer: SimpleEntity
    adults: number
    kids: number
    free: number
    totalPax: number
    totalEmbarked: number
    totalNoShow: number
    ports: LedgerPortGroupVM[]
    reservations: LedgerReservationVM[]

}

File: API/Features/Reservations/Ledgers/ViewModels/Report/LedgerVM.cs
using System.Collections.Generic;
using API.Infrastructure.Classes;

namespace API.Features.Reservations.Ledgers {

    public class LedgerVM {

        public SimpleEntity Customer { get; set; }
        public int Adults { get; set; }
        public int Kids { get; set; }
        public int Free { get; set; }
        public int TotalPax { get; set; }
        public int TotalEmbarked { get; set; }
        public int TotalNoShow { get; set; }
        public IEnumerable<LedgerPortVM> Ports { get; set; }
        public List<LedgerReservationVM> Reservations { get; set; }

    }

}